configure.py

```diff
@@ -42,10 +42,78 @@
 from multiprocessing import Pool
 from subprocess import run
 
+_CPU_FAMILY_MAP = {
+    "arm": "arm",
+    "aarch64": "arm",
+    "x86_64": "x86",
+}
+
+_ANDROID_VERSION = 28
+
+_ANDROID_COMPILER_MAP = {
+    "arm": f"armv7a-linux-androideabi{_ANDROID_VERSION}",
+    "aarch64": f"aarch64-linux-android{_ANDROID_VERSION}",
+    "x86_64": f"x86_64-linux-android{_ANDROID_VERSION}",
+}
+
+_ANDROID_ABI_MAP = {
+    "arm": "armeabi-v7a",
+    "aarch64": "arm64-v8a",
+    "x86_64": "x86_64",
+}
+
+_ANDROID_BUILD_MAP = {
+    "Linux": "linux-x86_64",
+    "Darwin": "darwin-x86_64",
+    "Windows": "windows-x86_64",
+}
+
+_ANDROID_CROSS_FILE_TPL = """\
```

Comment:
style: can you use `textwrap.dedent` so that the content can be indented away from column 0?

Reply:
Sure. Updated.